packages/ui/components/Summary.tsx

```diff
@@ -0,0 +1,3 @@
+import { ChakraComponent, chakra } from '@chakra-ui/react';
+
+export const Summary: ChakraComponent<'summary'> = chakra('summary');
```

Comment:
Many things inside packages/ui are components. I think this could go inside packages/ui/misc rather than creating a new subdirectory. If we end up wanting to keep useDetails

Reply:
Ah forgot about ui/misc, thanks